articles/connections/passwordless/configure-passwordless-connection.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Configure Passwordless Connections
+topics:
+    - connections
+    - passwordless
+    - sms
+    - email
+contentType: how-to
+useCase: customize-connections
+---
+# Configure Passwordless Connections
+
+This guide will show you how to configure Passwordless connections using Auth0's Dashboard. This task can also be performed using the Management API.
+
+## Configure Passwordless using SMS messages
+
+::: panel Twilio Account
+Before proceeding, you will need a valid Twilio account. Auth0 will use your [Twilio Account SID](https://www.twilio.com/help/faq/twilio-basics/what-is-an-application-sid) and [ Auth Token](https://www.twilio.com/help/faq/twilio-basics/what-is-the-auth-token-and-how-can-i-change-it) to send SMS messages tousers.
+:::
+
+1. Navigate to the [Connections > Passwordless](${manage_url}/#/connections/passwordless) page in the [Auth0 Dashboard](${manage_url}/), and enable the SMS toggle.
+
+![Enable SMS Passwordless](/media/articles/connections/passwordless/connections-passwordless-list.png)
+
+2. Enter your **Twilio Account SID** and **Twilio Auth Token**.
+
+::: note
+To learn how to find your Twilio SID and Auth Token, see the Twilio docs on the[Application SID](https://support.twilio.com/hc/en-us/articles/223136607-What-is-an-Application-SID-) and the [AuthToken](https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them).
+:::
+
+![Configure SMS Passwordless](/media/articles/connections/passwordless/connections-passwordless-sms.png)
+
+3. Select your **SMS Source** and, depending on your selection, enter either your **Twilio Messaging Service SID** or a **From** phone number. Users will see what you enter as the sender of the SMS.
+
+::: note
+To learn about using Twilio Copilot, see [Sending Messages with Copilot](https://www.twilio.com/docs/api/rest/sending-messages-copilot).
```

Comment:
Pretending to be a user: since I have no idea what Twilio Copilot is, I have no idea what this note is doing in this step, why I care, or if it's important. Maybe a tiny bit more context?